src/utils/shared/sidebar.js

```diff
@@ -77,14 +77,20 @@ function findPrevItemWithSource(data, item) {
 function normalizeItem({ rawItem, parentPath, resultRef, prevRef }) {
   validateRawItem(rawItem)
 
-  const { label, slug, source, tutorials, type, url } = rawItem
+  const { label, slug, source, tutorials, type, url, style, icon } = rawItem
```

Comment:
May want to update the inline doc example in sidebar.js

Reply:
*in the top of this file

Reply:
👍 yep

Reply:
@jorgeorpinel I'm not sure what you mean by inline doc example in this file. Did you mean to mention a doc file that describes how to use the sidebar?

Reply:
Done! #1332